analysis/paper/paper.Rmd

```diff
@@ -40,15 +40,15 @@ These are our notes for preparing our slides for our CAA2018 presentation in the
 
 ## Slide 1: Title slide    
 
-In this presentation we will talk about how archaeologists are changing their data analysis practices to make their research more reproducible and open. We will the rrtools package, which is our specific contribution to help with the challenge of making reproducible research in archaeology easier.
+In this presentation we will talk about how archaeologists are changing their data analysis practices to make their research more reproducible and open. We will introduce the rrtools package, which is our specific contribution to help with the challenge of making reproducible research in archaeology easier. "Our" meaning in this case a group of people whose names you can see here and / or who took part in thethe 2017 Summer School on Reproducible Research in Landscape Archaeology at the Freie Universität Berlin (17-21 July), funded and jointly organized by Excellence Cluster 264 Topoi, the Collaborative Research Center 1266, and ISAAKiel under the guidance of Ben Marwick. 
 
 ## Slide 2: Motivation
 
 Our motivation for this topic comes from a profound change that we have been observing in how researchers analyse their data.
 
 Increasingly, we are seeing researchers publishing research that uses a programming language to analyse their data. Similarly, we see journal articles accompanied by scripts of programming language so that readers can study the details of the data analysis methods. 
 
-This is an important change away from using mouse-driven, point-and-click software for data analysis. When archaeologists use software such as SPSS and Past (as you see in our screenshots here), it is very difficult for them to share the details of their analysis. When readers cannot see the details of an analysis, it is difficult for them to be confident in the results described in an article. 
+This is an important change away from using mouse-driven, point-and-click software for data analysis. When archaeologists use software such as SPSS and Past (as you see in our screenshots here), it is very difficult for them to share the details of their analysis. When readers cannot see the details of an analysis, it is difficult for them to be confident in the results described in an article. Also, for other researchers it may be near impossible to implement the described analysis in their own work as they lack the details on how to come to the desired conclusions. 
```

Comment:
Yes, great point!

Reply:
Also I noticed, going through the slides, that you're catchy phrases "expired", "tired" and "wired" are not once mentioned in our talk. I've been taught to never put something on a slide that's not mentioned in the talk. So I suggest we could add a picture with a hand-made seriation instead of one of the screen shots (I've got one from one of the introductory statistics texts) and then we could comment on these "stages" of development, just in a half-sentence: "to put this development in a catchy phase: ...". What do you think?

Reply:
That is a good point about mentioning slide contents. Since we are a bit overtime, how about we replace that little text book with the text box of the outline of our slides?

One the bottom of the summary slide we have tired-wired-inspired, which I think is ok, because that's a type of summary of our main point. And since it's on the last slide it will be easy for people to ask what you mean in the Q&A, if they are interested.

Reply:
Perhaps you can put the seriation figure after the last slide, and then if you have a chance during the Q&A you can show it and describe

Reply:
Yeah, that's a good idea. Let's not add any more content... ^^

Reply:
What do you think about deleting the slide on 'advanced reproducibliity'? We can just say that those are advanced topics that we will discuss at another time. That might help to shave a minute off

Reply:
I actually thought the same. I mean, it's great we have it, but most people may not even know what Markdown is (I thought about maybe adding a sentence there), so Docker might completely go over their heads. Or do you expect too little knowledge of the CAA guys? I really don't know what the level of prior knowledge to expect.

Reply:
Ok, great, yes, it's hard to know about the expertise of the audience. Probably there will be people in the room in both groups, know-nothing and know-all. My preference is to speak more to the know-nothings, since that is more inclusive. Plus the know-alls can be very opinionated and set in their ways which can be a little unpleasant to confront. 

I will move the 'advanced reproduciblity' slide to the end, and put the speaking text for that slide at the end of the document, and hide it in comments (this is the comment marker <!-- this text will be hidden in the HTML output --> ). So we wont see it in the nice output from the Rmd, but it's still in the Rmd in case we want to use it for something else.

Reply:
I agree with speaking at a non-expert level, anything else easily gets condescending. Ok, yeah that's good. I'll try my hand at compressing the text a bit in a new branch, see whether I can gain another minute.